christfellowshipchurch/web-app-v2
# Group Search Bug 2.0

### About
This PR fixes a bug in group search where the search reloads and the user is brought back to the top of the first page of the search every time a modal is closed.

### Test Instructions
1. Go to the group search page on our website ([here](https://www.christfellowship.church/groups/search?text=)) and testing opening any modal and closing it, it should reset the search and bring you back to the top of the first search page.
2. Repeat the last instruction, but on the testing link ([here](https://web-app-v2-git-page-not-stickin-c077e2-christ-fellowship-church.vercel.app/groups/search?text=)).
3. Test that search and filters work as intended.

### Screenshots
<img width="1439" alt="Screenshot 2023-09-01 at 9 30 19 AM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/c4476d5d-c3c0-4411-8900-a25d8c00fb30">


### Closes Tickets
[CFDP-2712](https://christfellowshipchurch.atlassian.net/browse/CFDP-2712)

[CFDP-2712]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
